### DISCUSSION TITLE
XAML Hot Reload in Jetbrains Rider?

### DISCUSSION THREAD

**Discussion by shingming** [unoplatform/uno#8924] [General]
Hi, I know the UNO Hot Reload feature now is supported in Visual Studio 2019, 2022 (Windows), and VS Code. Is there any planning that provided the UNO Hot Reload feature on Jetbrains Rider at macOS?

**Comment by jeromelaban** [comment]
Thanks for the question. We do not yet have Rider integration with Hot Reload, code completion at this time. We have plans to include support for those, but we do not know when it will be available.
